pyuavcan/transport/serial/_frame.py

```diff
@@ -131,7 +131,10 @@ def parse_from_cobs_image(image: memoryview) -> typing.Optional[SerialFrame]:
             unescaped_image = cobs.decode(bytearray(image))  # TODO: PERFORMANCE WARNING: AVOID THE COPY
         except cobs.DecodeError:
             return None
-        return SerialFrame.parse_from_unescaped_image(memoryview(unescaped_image))
+        try:
+            SerialFrame.parse_from_unescaped_image(memoryview(unescaped_image))
```

Comment:
`return` statement is missing, but the more important thing is that this exception block seems to be a little out of place: the docs for `parse_from_unescaped_image()` state that it should return None for invalid inputs but the actual behavior is that it throws. Should we not move the exception block there?